modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -371,198 +371,18 @@ var myDate = |2003-10-01T23:57:59.700-03:00|
 }
 ----
 
-[[date_format_change]]
-=== Changing the Format of a Date
+[[date_format]]
+=== Date and Time Formats
 
+To enable you to format dates and times, DataWeave supports formatting characters, such as the `u` (for the year), `M`, and `d` in the date format `uuuu-MM-dd`. These characters are based on the Java 8 `java.time.format` package.
 
-To format dates and times, DataWeave supports some formatting characters
```

Comment:
moved to modules/ROOT/pages/dataweave-cookbook-format-dates.adoc. Examples in the includes for script (modules/ROOT/pages/_partials/cookbook-dw/format-dates-ex03/transform.dwl) and 
for output (modules/ROOT/pages/_partials/cookbook-dw/format-dates-ex03/out.json)